src/main/resources/templates/newOrder.html

```diff
@@ -13,7 +13,7 @@
         };
     </script>
 
-    <p th:text="${order}" />
+    <p th:text="${Order}" />
```

Comment:
1) Covered in a previous code review
2) order is correct, per lowerCamelCase convention.